index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="./styles/index.css">
+</head>
+
+<body>
+  <header class="header">
+    <div class="header__first-nav">
+      <h3 class="header__logo">Medical Core</h3>
+      <nav class="header__links-container">
+        <a href="#" class="header__link">Home</a>
+        <a href="#" class="header__link">Product</a>
+        <a href="#" class="header__link">Pricing</a>
+        <a href="#" class="header__link">Contact</a>
+      </nav>
+    </div>
+    <nav class="header__auth-links">
+      <a href="#" class="header__link header__link--green">Login</a>
+      <button class="button button--primary">JOIN US</button>
+    </nav>
+  </header>
+  <main>
```

Comment:
Elements with no classes are not necessary in BEM.

Reply:
Thanks will work on that!

Reply:
Okay, sure. Don't forget to rerequest review once done implementing the feedback.

Reply:
Okay!